src/locale/locales/es/messages.po

```diff
@@ -73,112 +73,112 @@ msgstr ""
 #: src/components/ProfileHoverCard/index.web.tsx:398
 #: src/screens/Profile/Header/Metrics.tsx:23
 msgid "{0, plural, one {follower} other {followers}}"
-msgstr ""
+msgstr "{0, plural, un {follower} unos {followers}}"
 
 #: src/components/ProfileHoverCard/index.web.tsx:402
 #: src/screens/Profile/Header/Metrics.tsx:27
 msgid "{0, plural, one {following} other {following}}"
-msgstr ""
+msgstr "{0, plural, un {following} unos {following}}"
 
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:312
 msgid "{0, plural, one {Like (# like)} other {Like (# likes)}}"
-msgstr ""
+msgstr "{0, plural, un {Like (# like)} unos {Like (# likes)}}"
```

Comment:
```suggestion
msgstr "{0, plural, one {Me gusta (# me gusta)} other {Me gusta (# me gusta)}}"
```
Twitter seems to use `Me gusta` for both singular and plural as the noun form of `like`, the existing Bluesky translation is a bit inconsistent. Does this suggestion make sense?